### PR TITLE
Header

### PR DESCRIPTION
**Pull request by Sarahshr**
Top of the homepage

### CODE REVIEW

File: src/Components/Header/Header.jsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import React, { useState } from "react";
+
+import Sidebar from "./Sidebar";
+import logo from "../../assets/images/logo.png";
+import {
+  Main,
+  HeaderBar,
+  Container,
+  H1,
```
Comment (AlexisCatrix) [inline src/Components/Header/Header.jsx]: Maybe use "Title" instead "H1" :  more explicit

File: src/Components/Header/Sidebar.jsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
+import {
+  Main,
+  IconBars,
+  Nav,
+  Ul,
```
Comment (AlexisCatrix) [inline src/Components/Header/Sidebar.jsx]: Not enough explicit

File: src/Components/Header/Sidebar.jsx [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  CloseIcon,
+  Li,
```
Comment (AlexisCatrix) [inline src/Components/Header/Sidebar.jsx]: Not enough explicit

**Review by AlexisCatrix — CHANGES_REQUESTED** [review]

**Review by AlexisCatrix — APPROVED** [review]
Good !

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**